(#1057) Add connection string info and update dependency information for CCM 0.13.0 (#1060)
chocolatey/docs: src/content/docs/en-us/central-management/setup/database.mdx

### Package Parameters

> * `/ConnectionString:` - The SQL Server database connection string to be used to connect to the Chocolatey Central Management database. Defaults to default or explicit values for `/SqlServiceInstance` and `/Database`, along with Integrated Security (`Server=<LOCAL COMPUTER FQDN NAME>; Database=ChocolateyManagement; Trusted_Connection=True; TrustServerCertificate=true;`). The account should have `db_owner` access to the database ([database owner](https://docs.microsoft.com/en-us/sql/relational-databases/security/authentication-access/database-level-roles#fixed-database-roles)).

Reviewer: > Defaults to default

What does that mean? Does it mean that it defaults to the word, `default`? If so, can we put code quotes around it?

That whole sentence really doesn't read very well. I know you didn't write it.

Author: 👀 

_Yeah_, I can reword that a bit, for sure. It's trying to say "the `ConnectionString` defaults to [a value that is constructed from the specified or default values for `/SqlServerInstance` and `/Database`, along with Integrated Security".

... Lemme workshop that a bit.

> * `/ConnectionString:` - The SQL Server database connection string to be used to connect to the Chocolatey Central Management database. Defaults to default or explicit values for `/SqlServiceInstance` and `/Database`, along with Integrated Security (`Server=<LOCAL COMPUTER FQDN NAME>; Database=ChocolateyManagement; Trusted_Connection=True; TrustServerCertificate=true;`). The account should have `db_owner` access to the database ([database owner](https://docs.microsoft.com/en-us/sql/relational-databases/security/authentication-access/database-level-roles#fixed-database-roles)).

Reviewer: ```suggestion
* `/ConnectionString:` - The SQL Server database connection string to be used to connect to the Chocolatey Central Management database. Defaults to default or explicit values for `/SqlServiceInstance` and `/Database`, along with Integrated Security (`Server=<LOCAL COMPUTER FQDN NAME>; Database=ChocolateyManagement; Trusted_Connection=True; TrustServerCertificate=true;`). The account should have [`db_owner`](https://docs.microsoft.com/en-us/sql/relational-databases/security/authentication-access/database-level-roles#fixed-database-roles) access to the database.

```

> * `/ConnectionString:` - The SQL Server database connection string to be used to connect to the Chocolatey Central Management database. Defaults to default or explicit values for `/SqlServiceInstance` and `/Database`, along with Integrated Security (`Server=<LOCAL COMPUTER FQDN NAME>; Database=ChocolateyManagement; Trusted_Connection=True; TrustServerCertificate=true;`). The account should have `db_owner` access to the database ([database owner](https://docs.microsoft.com/en-us/sql/relational-databases/security/authentication-access/database-level-roles#fixed-database-roles)).

Reviewer: ```suggestion
* `/ConnectionString:` - The SQL Server database connection string to be used to connect to the Chocolatey Central Management database. Defaults to default or explicit values for `/SqlServiceInstance` and `/Database`, along with Integrated Security (`Server=<LOCAL COMPUTER FQDN NAME>; Database=ChocolateyManagement; Trusted_Connection=True; TrustServerCertificate=True;`). The account should have `db_owner` access to the database ([database owner](https://docs.microsoft.com/en-us/sql/relational-databases/security/authentication-access/database-level-roles#fixed-database-roles)).

```

Reviewer: We should be consistent.

        <Callout type="warning">

>             **Installs**: Please ensure the user running the package installation is able to create databases unless you also pass `/SkipDatabasePermissionCheck` (in that case you simply need `db_owner` to the database being managed if it was pre-created).

Reviewer: ```suggestion
            **Installation**: Please either ensure the user running the package installation is able to create databases or pass `/SkipDatabasePermissionCheck`, if you have `db_owner` permission to the pre-created database.
```

>             **Upgrades**: Please ensure the user running the package installation has been granted `db_owner` access to an existing database.

Reviewer: ```suggestion
            **Upgrade**: Please ensure the user running the package installation has been granted `db_owner` access to the existing database.
```

        <Callout type="warning">

>             **Installs**: Please ensure the login credentials provided are able to create databases unless you also pass `/SkipDatabasePermissionCheck` (in that case you simply need `db_owner` to the database being managed if it was pre-created).

Reviewer: ```suggestion
            **Installation**: Please either ensure the user running the package installation is able to create databases or pass `/SkipDatabasePermissionCheck`, if you have `db_owner` permission to the pre-created database.
```

>             **Upgrades**: Please ensure the login credentials provided have been given `db_owner` access to an existing database.

Reviewer: ```suggestion
            **Upgrade**: Please ensure the user running the package installation has been granted `db_owner` access to the existing database.
```

```json
{
  "ConnectionStrings": {

>     "Default": "Server=<HOST_NAME_OF_MACHINE_BEING_INSTALLED_ONTO>; Database=ChocolateyManagement; Trusted_Connection=True; TrustServerCertificate=true;"

Reviewer: ```suggestion
    "Default": "Server=<HOST_NAME_OF_MACHINE_BEING_INSTALLED_ONTO>; Database=ChocolateyManagement; Trusted_Connection=True; TrustServerCertificate=True;"

```

Reviewer: We should be consistent.

>     "Default": "Server=<HOST_NAME_OF_MACHINE_BEING_INSTALLED_ONTO>; Database=ChocolateyManagement; Trusted_Connection=True; TrustServerCertificate=True;"

Reviewer 2: I really wish I could comment on the line I want to... The line above  indicates that the default is as of `0.6.2`... Should that be updated? Or perhaps just removed?

Perhaps someting like "The currently supported version of Chocolatey Central Management contains these defaults" 🤔

Author: I think taking out the version number makes sense here, yep.

Author: Hmm. Actually, I'm not sure. I think a "since 0.13.0" might be an alright thing, since it will make it a little clearer for any folks who are insisting on remaining on an older version for whatever reason.
